fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/block/FabricBlockSettings.java

```diff
@@ -97,56 +91,72 @@ protected FabricBlockSettings(AbstractBlock.Settings settings) {
 		this.postProcess(otherAccessor.getPostProcessPredicate());
 	}
 
+	/**
+	 * @deprecated replace with {@link AbstractBlock.Settings#create()}
+	 */
+	@Deprecated
 	public static FabricBlockSettings create() {
 		return new FabricBlockSettings();
 	}
 
 	/**
-	 * @deprecated Use {@link FabricBlockSettings#create()} instead.
+	 * @deprecated replace with {@link AbstractBlock.Settings#create()}
 	 */
 	@Deprecated
 	public static FabricBlockSettings of() {
 		return create();
 	}
 
+	/**
+	 * @deprecated replace with {@link AbstractBlock.Settings#copy(AbstractBlock)}
+	 */
 	public static FabricBlockSettings copyOf(AbstractBlock block) {
```

Comment:
This should include the `@Deprecated` annotation.
```suggestion
	@Deprecated
	public static FabricBlockSettings copyOf(AbstractBlock block) {
```